docs/streaming/sources/kafka.mdx

```diff
@@ -14,8 +14,30 @@ include_metadata: false
 api_version: "0.10.2"
 batch_size: 100
 timeout_ms: 500
+auto_offset_reset: latest # Optional
```

Comment:
what are the possible values for auto_offset_reset?

Reply:
There are only 2 possible values: latest & earliest
If the consumer faces a issue reguarding offsets ( e.g OffsetOutOfRange), the consumer will reset its offset based on the given value.
earliest meaning the oldest message possible
latest meaning the most recent message possible